plugins/node/opentelemetry-instrumentation-pg/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@opentelemetry/instrumentation-pg",
+  "version": "0.13.1",
+  "description": "OpenTelemetry postgres automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "npm run version:update && tsc -p .",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test:debug": "ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/**/*.test.ts'",
+    "test:local": "cross-env RUN_POSTGRES_TESTS_LOCAL=true npm run test",
+    "version:update": "node ../../../scripts/version-update.js",
+    "watch": "tsc -w"
+  },
+  "keywords": [
+    "opentelemetry",
+    "postgres",
+    "pg",
+    "postgresql",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "plugin",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
```

Comment:
is it intended to use `files` in package.json and a .npmignore file? Usually I would expect only one variant to filter the files to be included.

Reply:
Check all of our plugins and instrumentations they looks the same. If you think it should be changed then I guess there should be a separate issue where you can propose the solution, but currently all our packages looks like this